sycl/source/detail/scheduler/commands.cpp

```diff
@@ -1270,6 +1283,10 @@ ContextImplPtr MemCpyCommandHost::getContext() const {
   return detail::getSyclObjImpl(Queue->get_context());
 }
 
+QueueImplPtr MemCpyCommandHost::getWorkerQueue() const {
+  return MQueue->is_host() ? MSrcQueue : MQueue;
+}
+
 cl_int MemCpyCommandHost::enqueueImp() {
   QueueImplPtr Queue = MQueue->is_host() ? MSrcQueue : MQueue;
```

Comment:
NIT: Should we use `getWorkerQueue()` where possible now?

Reply:
Done